src/opnsense/mvc/app/controllers/OPNsense/Diagnostics/Api/InterfaceController.php

```diff
@@ -74,7 +74,18 @@ public function getArpAction()
 
         return $arptable;
     }
-
+    
+    /**
+     * retrieve system arp table contents
+     * @return array
+     */
+    public function flushArpAction()
+    {
```

Comment:
@peppelinux please validate for a post type request, and use ajaxCall in the volt template. this breaks csrf.

Reply:
You're right!
I've just cloned the programming style of InterfaceController.php.
By the way, Are you asking me to put something like:
````
if ($this->request->isPost()) { ...
````
if it is it consider it done but is there any suggestion on csf token variable name to use in the template?

Reply:
csrf headers are included and validated automatically,  the isPost() is indeed what I mean.

Reply:
Fixed :+1: